File: Python_Test/Chips.py
from Main import getInput

playerChips = 0
playerBet = 0

def addChips():
    while (True):
        try:
            amount = getInput("How many chips to add: ")
            if(amount <= 0):
                raise ValueError()
            else: 
                break
        except ValueError:
            print("Invalid input. Please enter a valid number")

def newBet():
    while(True):
        amount = getInput("How many chips to bet: ")
        if (playerChips < amount):
            print("You don't have enough chips.\n")
            addChips()
        elif (playerChips == 0):
            print("You don't have any chips.\n")
        else:
            playerBet = amount
            break

def getPlayerChips():
    return playerChips

def setChips(amount: int, operation: str):
    if(operation == "+"):
        #playerChips += amount
        return

File: Older/second-version/Chips.java
package version2;
public class Chips {
    public static int playerChips = 250;
    public static int playerBet = 0;

    public static void addChips() {
        int amount = Main.getInput("How many chips to add: ");
        playerChips += amount;
    }

    public static void newBet() {
        while (true) {
            int amount = Main.getInput("How many chips to bet: ");
            if (playerChips < amount) {
                System.out.println("You don't have enough chips.\n");
                addChips();
            } else if (playerChips == 0) {
                System.out.println("You don't have any chips.\n");
                addChips();
            } else {
                playerBet = amount;
                break;
            }
        }
    }
}